packages/components/src/Combobox/Combobox.types.ts

```diff
@@ -58,6 +58,11 @@ export interface ComboboxProps {
    * Should the Combobox display the loading state.
    */
   readonly loading?: boolean;
+
+  /**
+   * A ref to the default activator button element.
+   */
+  readonly defaultActivatorRef?: React.RefObject<HTMLButtonElement>;
```

Comment:
Minor comment, I think this should should `React.Ref<HTMLButtonElement>` because the ref might be a callback ref (this is what RHF uses). That way consumers don't need to cast it

Reply:
that's fair. I'm fine to change that here if it's the more flexible implementation

though worth noting that's not typically what we use, so we might need to make some updates to other components someday

![image](https://github.com/user-attachments/assets/0a2ca29c-a27e-4cb5-8f18-64c9ff966c6a)